src/helperComponents/PropertiesDialog/GenbankView.js

```diff
@@ -13,6 +13,9 @@ class GenbankView extends React.Component {
   state = {
     fileTypeToView: "genbank"
   };
+
+  select = () => this._input?.select();
```

Comment:
this change fixed the Genbank View selection and also the console warning in dev tool.